templates/.kokoro/release/publish_javadoc11.cfg

```diff
@@ -0,0 +1,30 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# cloud-rad production
+env_vars: {
+  key: "STAGING_BUCKET_V2"
+  value: "docs-staging-v2-dev"
```

Comment:
Using dev bucket

Reply:
Can you add a comment in the file itself to remind us what to change it to for prod?